### ISSUE TITLE
Bug: Failed prop type console warning in backend

### ISSUE THREAD

**Issue by GuangHa** [neos/neos-ui#3481]
### Description

When the Neos-Backend is opened a console error occurs. The error does not occur when you switch to another node via the document tree, it happens when you reload the page.
This happens since the last Neos-Patch-Update (Possibly this package from 7.3.12 to 7.3.17).

### Steps to Reproduce

1. Open Neos-Backend
2. Console warning/error occurs

#### Expected behavior

The error/warning does not occur.

#### Actual behavior

The error/warning occurs:
![image](https://user-images.githubusercontent.com/12826344/235863024-2b608a07-668b-4fa2-977a-9b2d2a199dd7.png)

### Affected Versions

Neos: 7.3.12
UI: 7.3.17
PHP: 8.1.2

### Additional Information

Tested on Firefox and Chrome
Firefox: 112.0
Chrome: 112.0.5615.165

Packages in composer.json which are not custom created:
```"neos/neos": "~7.3.10",
"neos/nodetypes": "~7.3.0",
"neos/seo": "~3.2",
"neos/setup": "^5.0",
"neos/redirecthandler-neosadapter": "^4.3",
"neos/redirecthandler-databasestorage": "^5.0",
"sitegeist/monocle": "~7.0",
"flowpack/cachebuster": "^1.1",
"neos/swiftmailer": "^7.2",
"flowpack/nodetemplates": "^1.3",
"moc/imageoptimizer": "^4.0",
"sitegeist/magicwand": "~4.4",
"neos/redirecthandler-ui": "^2.3",
"neos/neos-ui": "~7.3.0"

**Comment by Sebobo** [comment]
Did you set the the `isHidden` value of a nodetype property somewhere to a string?
Like with `ClientEval: xyzzy` or similar?

**Comment by GuangHa** [comment]
No, I searched all my packages with "isHidden" or "ClientEval", neither of them is used in the code.

**Comment by markusguenther** [comment]
@GuangHa While we couldn't reproduce the reported issue, we want to ensure a seamless experience for all users. If this problem persists on your end or if you have additional information, could you kindly let us know? Your assistance is highly appreciated!
